src/platform/packages/shared/kbn-management/settings/setting_ids/index.ts

```diff
@@ -59,6 +59,9 @@ export const LABS_DASHBOARDS_ENABLE_UI_ID = 'labs:dashboard:enable_ui';
 // Accessibility settings
 export const ACCESSIBILITY_DISABLE_ANIMATIONS_ID = 'accessibility:disableAnimations';
 
+// Agent builder settings
+export const AGENT_BUILDER_ENABLED_SETTING_ID = 'agentBuilder:enabled';
```

Comment:
putting this here for the future, when we need to expose this in UI. So that it's in shared public package

Reply:
can you elaborate? why not add it then?